packages/lexical-devtools/src/panel/components/App/index.tsx

```diff
@@ -88,12 +94,16 @@ function App(): JSX.Element {
           <p>Loading...</p>
         </div>
       ) : (
-        <TreeView
-          deHighlightDOMNode={deHighlightDOMNode}
-          highlightDOMNode={highlightDOMNode}
-          viewClassName="tree-view-output"
-          nodeMap={nodeMap}
-        />
+        <>
+          <TreeView
+            deHighlightDOMNode={deHighlightDOMNode}
+            handleNodeClick={handleNodeClick}
```

Comment:
If the args are the same: 

```suggestion
            handleNodeClick={setSelectedNode}
```